core/src/ast/operator.rs

```diff
@@ -1,29 +1,48 @@
 use serde::{Deserialize, Serialize};
+use strum_macros::Display;
 
-#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, Display)]
 pub enum UnaryOperator {
+    #[strum(to_string = "+")]
     Plus,
+    #[strum(to_string = "-")]
     Minus,
+    #[strum(to_string = "~")]
     Not,
-    Factorial,
+    // #[strum(to_string="factorial")]
+    Factorial, // why is this here and not in the function enum?
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, Display)]
 pub enum BinaryOperator {
+    #[strum(to_string = "+")]
     Plus,
+    #[strum(to_string = "-")]
     Minus,
+    #[strum(to_string = "*")]
     Multiply,
+    #[strum(to_string = "/")]
```

Comment:
In this case, `strum_macros::Display` conversion is too radical.
How about.. rather than using Display, to impl `ToSql` trait for these (BinaryOperator and UnaryOperator)